includes/Modules/Tag_Manager.php

```diff
@@ -638,4 +640,25 @@ private function register_tag() {
 		}
 	}
 
+	/**
+	 * Can the Analytics module insert it's own snippet.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param boolean $original_value Original value of useSnippet setting.
+	 *
```

Comment:
This is a bit inconsistent in some places, but there shouldn't be a line between `@param` and `@return` annotations. (PHP linting is not as thorough as ESLint about this _yet_ 😄 )
```suggestion
```